packages/terra-form-select/src/Combobox.jsx

```diff
@@ -85,6 +85,10 @@ const propTypes = {
    * Whether the field is required.
    */
   required: PropTypes.bool,
+  /**
+   * Render dropdown menu in normal DOM flow with position absolute. Renders in a portal by default.
+   */
+  useSemanticDropdown: PropTypes.bool,
```

Comment:
Nominating a name change here. Names are challenging, but here are a few rough ideas.

- `disablePortal`
- `isPositionAbsolute`
- `isAbsolutePositioned`
- `isInline`
- `isDropdownInline`
- `isInlineDropdown`

`isPortaled` would be nice, but we want a variable that defaults to false. I don't particularly like any of these names, but maybe some combination of them will work.

Reply:
Yeah... Naming. What a constant battle. We started with isDropdownInline, but that was before we had position absolute'd the menu. So it made more sense... I chose useSemanticDropdown after we talked with UX on Monday since it is using semantic markup now and with the hope that the name would lead to teams wanting to use semantic markup...

Even thought about something like preferAccessibility to try to guilt people into opting in 😂

Reply:
I'm not sure most people will understand the semantic markup terminology. Whatever we choose I think accessibility needs to be called out in the code comments and prop documentation for clarity and reinforcement. 

Although it goes against the idea that all components should be accessible by default, I like `preferAccessibility` or `isAccessible` because it clearly says why I would care about this and then later when it is `true` it will be a code smell when people are using `isAccessible=false`. Much like how `dangerouslySetInnerHtml` is a cue that I'm doing something abnormal.

Reply:
Yeah, I do agree that "use semantic" will be lost on most consumers. I'm split on which one I like more: `preferAccessibility` or `isAccessible`

Reply:
@bjankord Crushing it. Love it. Sounds the most reasonable.

Reply:
+1